dbms/src/Flash/Coprocessor/DAGQueryBlockInterpreter.cpp

```diff
@@ -850,7 +870,10 @@ void DAGQueryBlockInterpreter::executeImpl(DAGPipeline & pipeline)
     // execute exchange_sender
     if (query_block.exchange_sender)
     {
-        handleExchangeSender(pipeline);
+        if (dagContext().isTest())
```

Comment:
Do we need `if (unlikely(dagContext().isTest()))`?
 What do you think about it?

Reply:
Good idea, I will fix them all.